qt/app.py

```diff
@@ -755,15 +751,34 @@ def _context_menu_toolbar_style(self,
                 _('Text beside icon'),
                 Qt.ToolButtonStyle.ToolButtonTextBesideIcon),
         )
-        for text, style in options:
+    
+    def _set_toolbar_button_style(self, toolbar, style, index):
+        """Set toolbar button style and store the selected index."""
+        toolbar.setToolButtonStyle(style)
+        self.toolbar_button_style = index
```

Comment:
The ToolButtonStyle's are just integers. See https://www.riverbankcomputing.com/static/Docs/PyQt6/api/qtcore/qt.html#ToolButtonStyle

`enumerate()` makes it more complex than it need to be. Just cast them to integers before store them in the state file.

```suggestion
    def _set_toolbar_button_style(self, toolbar, style, index):
        """Set toolbar button style and store the selected index."""
        toolbar.setToolButtonStyle(style)
        self.toolbar_button_style = int(style)
```